### ISSUE TITLE
"auth_failed" error on automatic setup of Lock, and I don't understand what I have to do manually

### ISSUE THREAD

**Issue by imsandro** [home-assistant/home-assistant.io#25576]
### Feedback

I was trying to add the Lock, that already had been discovered automatically by HA, but I can't login to my account with my mail and password. Doesn't know what's wrong. It just gives me an error message saying "auth_failed".

So, I went to see how can I do it manually, but the problem is that I can't understand what I need to do based on the documentation - more specifically how to find the codes. I'm not a programmer, so I really would love some step by step guide to find the codes. The link you provided isn't helping a noob like me.

### URL

https://www.home-assistant.io/integrations/switchbot

**Comment by bdraco** [comment]
The login is case sensitive and will only work with an actual SwitchBot account (no third party auth provides like apple etc)

**Comment by fangoward** [comment]
@bdraco  yeah, I know. Mine was linked with Google and I just moved everything into a new account created from scratch.  But the problem still remains. Can't login even with a fresh account.

**Comment by bdraco** [comment]
Can you try to get a debug log?

https://www.home-assistant.io/docs/configuration/troubleshooting/#enabling-debug-logging

**Comment by fangoward** [comment]
I can't seem to find that option on the Lock that's being discovered.
This is what I have: https://prnt.sc/C63EXfYXnHxG
